save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ContestListView.kt

```diff
@@ -68,18 +93,63 @@ class ContestListView : AbstractView<ContestListViewProps, State>(false) {
                     }
                 }
             }
+            column(id = "enroll_button", header = "Registration", { name }) { cellProps ->
+                Fragment.create {
+                    td {
+                        button {
+                            className = ClassName("btn btn-primary")
+                            onClick = {
+                                setState {
+                                    selectedContestName = cellProps.value
+                                    isProjectSelectorModalOpen = true
+                                }
+                            }
+                            +"Participate in ${cellProps.value}"
```

Comment:
Isn't it too much text? Why not just `Participate`?

Reply:
Oh it seemed like more clear
Ok, will fix it